packages/api/README.md

```diff
@@ -21,6 +21,15 @@ workers_dev = true
 account_id = "<what does the `wrangler whoami` say>"
 ```
 
+- `npm run build` - Build the bundle
+- Add secrets
+
+    ```sh
+    wrangler secret put MAGIC_SECRET_KEY --env $(whoami) # Get from magic.link account
```

Comment:
I hit a problem setting up nft.storage where I could not set secrets before I had published the worker for the first time. It might have been a one off, but worth checking, or just move this step after the publish step to avoid the possibility.

Reply:
When I did this it attempted to create a worker for me, but needed a built bundle - thats why I added the `npm run build` step prior to this one.